src/components/ReviewCard.tsx

```diff
@@ -10,6 +12,19 @@ export type ReviewCardProps = {
 const ReviewCard = ({ review, isOwnReview }: ReviewCardProps) => {
   const { rating, createdAt, comment, imageId } = review;
 
+  const [thumbsignalData, setThumbsignalData] = useState<Thumbsignal | null>(
+    null,
```

Comment:
null should be used to say that this data has been fetched, but it is empty or does not exist, but before the fetching is done, it should be undefined. Null means that you know that the value is empty, when undefined could be empty, but it might just be because it is not yet populated.